libs/search/src/lib/state/actions.ts

```diff
@@ -21,6 +22,12 @@ export class SortBy implements Action {
   constructor(public sortBy: string) {}
 }
 
+export class UpdateResultsLayout implements Action {
+  readonly type = UPDATE_RESULTS_LAYOUT
```

Comment:
`SET_RESULTS_LAYOUT` is more appropriate

Reply:
Same for SET_SORT_BY instead of SORT_BY ?

Reply:
UPDATE_FILTERS ?

Reply:
Yes `SET_SORTBY`, we keep `UPDATE_FILTERS` cause the payload is a patch.